src/snowflake/snowpark/_internal/compiler/query_generator.py

```diff
@@ -151,7 +151,7 @@ def do_resolve_with_resolved_children(
                 copied_resolved_child,
                 logical_plan,
                 self.session._use_scoped_temp_objects,
-                logical_plan.is_generated,
+                logical_plan.creation_source,
```

Comment:
don't you also need to update the child_attributes to pass this as None if not exist?

Reply:
I was planning to make it part of large query breakdown. Would you say I add it in this one

Reply:
added in this PR